business-registry-dashboard/app/pages/incorporateNowFromNr.vue

```diff
@@ -77,9 +72,14 @@ onMounted(async () => {
       }
     }
 
+    // Load user's affiliated businesses and check if this name request is already affiliated
+    // This prevents unauthorized access to name requests that belong to other users
+    await affStore.loadAffiliations()
+    const isAffiliated = isNameRequestAffiliated(affStore.affiliations.results, business.nameRequest?.nrNumber)
```

Comment:
I think we want the NR to be affiliated as part of the magic link flow.

This flow is analogous to the "incorporate now" flow from an approved NR in Namerequest. Re: https://github.com/bcgov/namerequest/blob/main/app/src/mixins/nr-affiliation-mixin.ts

Reply:
By the way, when I tested the Namerequest flow, I deleted the draft IA and was left with the NR in my MBR. I think we want this behaviour here, too.

Reply:
That'll be fixed here: https://app.zenhub.com/workspaces/entities---olga-65af15f59e89f5043c2911f7/issues/gh/bcgov/entity/24885

Reply:
Sorry, I'm saying something else.

I'm saying that Namerequest UI affilaites the NR first, and then creates a draft IA, so that if the user deletes the draft IA, they are left with the NR in their MBR. This is good. Ideally the magic incorporation link process would work the same way.

Reply:
New flow looks great. And thanks for the super-useful comment in the code.